codecs/rotate/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "rotate",
+  "scripts": {
+    "build": "mv rotate.{as,ts} && asc rotate.ts -b rotate.wasm --validate -O3 && mv rotate.{ts,as}"
```

Comment:
Does asc depend on the extension?

Reply:
Currently it uses only `.ts`.

Reply:
you can move as code inside `assembly` directory and add it to exclude in root `tsconfig.json`